include/cuter_macros.hrl

```diff
@@ -44,18 +44,76 @@
 -define(VISITED_TAGS_PREFIX, '__visited_tags').
 -define(EXECUTION_COUNTER_PREFIX, '__exec_counter').
 
+%%====================================================================
+%% Configuration parameters.
+%%====================================================================
+
+% If enabled, we keep execution traces for debugging.
```

Comment:
All these comments should start with `%%` (for emacs).

Reply:
Done.